airbyte-integrations/connectors/destination-pinecone/metadata.yaml

```diff
@@ -12,14 +12,14 @@ data:
   connectorSubtype: database
   connectorType: destination
   definitionId: 3d2b6f84-7f0d-4e3f-a5e5-7c7d4b50eabd
-  dockerImageTag: 0.0.6
+  dockerImageTag: 0.0.7
   dockerRepository: airbyte/destination-pinecone
   githubIssueLabel: destination-pinecone
   icon: pinecone.svg
   license: MIT
   name: Pinecone
   releaseDate: 2023-08-15
-  releaseStage: alpha
+  releaseStage: beta
```

Comment:
could you also add the `ab_internal` flag? https://github.com/airbytehq/airbyte/blob/ac56c5cc14c208da25a2becadfecda0261ec58d7/airbyte-integrations/connectors/source-s3/metadata.yaml#L28